lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -248,6 +248,35 @@ function dosomething_northstar_save_id_field($uid, $northstar_response) {
 }
 
 /**
+ * Save the user's Northstar ID to their local profile.
+ *
+ * @param object $account
+ *   Fully loaded user object.
+ *
+ * @return string
+ *   The Northstar id or empty string on error.
+ */
+function dosomething_northstar_get_id($account = NULL) {
```

Comment:
hmm, the module isn't turned off on stage, only locally, and there should always be values there locally b/c you pull the database from stage, that shouldn't fail